.github/workflows/qa-deploy.yml

```diff
@@ -0,0 +1,36 @@
+name: RNS QA build and deploy
+
+on:
+  push:
+    branches: [ disposable-main ]
+
+jobs:
+  qa_build_and_deploy:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup Node 12
+        uses: actions/setup-node@v1
+        with:
+          node-version: '12'
+
+      - name: Install dependencies
+        run: yarn
+
+      - name: Build site
+        run: yarn build:mainnet
```

Comment:
Shouldn't we use `yarn:build:testnet` for QA?

Reply:
Since we do not have the source code for the old version of the smart contracts, we can't have a testnet environment, that is why we'll have to use the mainnet version. However if we found a work around this, I agree it is better to have a testnet version for QA.